ALU_wrapper.py

```diff
@@ -0,0 +1,128 @@
+#!/apps/bio/software/anaconda2/envs/hcp/bin/python3
+
+# Please note, this is an early version. All the verbose-stuff will be reduced at later stages.
+# TODO: functions for copy finished data to web folder
+# TODO: Information to slims?
+# TODO: What to do with original folders?
+
+import os
+import glob
+import smtplib
+import argparse
+import subprocess
+from email.message import EmailMessage
+
+mail_to = ""  # Change this, and uncomment argument before EOF
+pythonpath = "/apps/bio/software/anaconda2/envs/hcp/bin/python3"  # since server (as for now) has python2 installed
+
+parser = argparse.ArgumentParser(description='Wrapper to find non-processed folders, run ALU_parser on them.')
+parser.add_argument("-p", "--path", help="Path to folder with sequence-subfolders.", action="store", default=".")
+# /seqstore/remote/share/crc
+parser.add_argument("-r", "--run", help="Run for real. If not specified, just dry-run without any changes.",
+                    action="store_true")
+parser.add_argument("-v", "--verbose", help="Show more information while running.", action="store_true")
+args = parser.parse_args()
+sendmail = False
```

Comment:
I guess this should be an arg-parse flag?

Reply:
Since the mail will be sent (or not) depending on a boolean check, a good practice is to declare and initialize the variable before it is used in the code. In this case, the code may run but not find any files to process, and then no mail should be sent. We could move the mail-part to the parsing-part (or similar location), and avoid the need for the check completely. But if we want to force the script to mail... a flag could be used for this? Sorry, I am a bit tired, perhaps I am missing something here. Or did you mean that perhaps we do not want it to mail regardless of parsing files or not? An override-flag perhaps?